src/GeoExt/data/FeatureStore.js

```diff
@@ -245,9 +249,13 @@ Ext.define('GeoExt.data.FeatureStore', {
      * @return {String} The model instance corresponding to a feature.
      */
     getByFeature: function(feature) {
-        return this.getAt(this.findBy(function(record, id) {
-            return record.raw == feature;
-        }));
+        var featureKey = GeoExt.isExt4 ? 'raw' : 'data',
+            comparisonFunc = function(record, id) {
```

Comment:
There is a comma (`,`) in the previous line, so this should not introduce an unexpected global, right?